pkg/plugin/plugin.go

```diff
@@ -268,16 +284,6 @@ func WaitForReady(client pb.KeyManagementServiceClient, duration time.Duration)
 	return nil
 }
 
-// Check validates the availability of the server using the provided client
-func Check(client pb.KeyManagementServiceClient) (string, error) {
```

Comment:
i didnt find a removed reference. Was this method not used?

Reply:
Not used anywhere.